modules/_labs/breadcrumbs/react/lib/Breadcrumbs/Dropdown/Menu.tsx

```diff
@@ -0,0 +1,116 @@
+/** @jsx jsx */
+import React from 'react';
+import {css, jsx} from '@emotion/core';
+import {
+  borderRadius,
+  colors,
+  commonColors,
+  depth,
+  spacing,
+  type,
+} from '@workday/canvas-kit-react-core';
+
+// types
+import {DropdownContext} from './Provider';
+import {Breadcrumb} from '../types';
+// local components
+import {DropdownMenuItemLink} from './MenuItemLink';
+
+export interface DropdownMenuProps
+  extends React.HTMLAttributes<HTMLUListElement>,
+    Pick<DropdownContext, 'activeDropdownItem' | 'dropdownItems' | 'setActiveDropdownItem'> {
+  activeDropdownItemRef: React.RefObject<HTMLAnchorElement>;
+  resetFocus: () => void;
+  toggleActiveItemUp: () => void;
+  toggleActiveItemDown: () => void;
+}
+
+const menuWrapperStyles = css({
+  ...type.body,
+  ...depth[2],
+  backgroundColor: commonColors.background,
+  border: `1px solid ${colors.soap500}`,
+  borderRadius: borderRadius.m,
+  boxSizing: 'border-box',
+  display: 'inline-block',
+  marginTop: spacing.xxxs,
+  position: 'relative',
+  width: 'max-content',
+});
+
+const MenuWrapper = (props: React.HTMLAttributes<HTMLDivElement>) => {
+  return <div css={menuWrapperStyles} {...props} />;
+};
+
+const menuStyles = css({
+  boxSizing: 'border-box',
+  listStyle: 'none',
+  margin: `${spacing.xxs} 0`,
+  padding: 0,
+  width: 280,
+  wordBreak: 'break-word',
+});
+
+const Menu = (props: React.HTMLAttributes<HTMLUListElement>) => {
+  return <ul role="menu" css={menuStyles} {...props} />;
+};
+
+export const DropdownMenu = ({
+  activeDropdownItemRef,
+  activeDropdownItem,
+  dropdownItems = [],
+  resetFocus,
+  setActiveDropdownItem,
+  toggleActiveItemDown,
+  toggleActiveItemUp,
+  ...props
+}: DropdownMenuProps) => {
+  const handleItemKeyUp = (e: React.KeyboardEvent<HTMLLIElement>) => {
+    switch (e.key) {
+      case 'ArrowUp':
+      case 'ArrowLeft':
+      case 'Up': // IE/Edge specific value
+        return toggleActiveItemUp();
+      case 'ArrowDown':
+      case 'Down': // IE/Edge specific value
+      case 'ArrowRight':
+        return toggleActiveItemDown();
+      default:
+        break;
+    }
+  };
+
+  const handleItemKeyDown = (e: React.KeyboardEvent<HTMLLIElement>, item: Breadcrumb) => {
+    switch (e.key) {
+      case 'Tab':
+        e.preventDefault();
+        break;
+      case 'Enter':
+        e.preventDefault();
+        if (item.onAction) {
+          item.onAction(item);
+          return resetFocus();
+        }
+        return (window.location.href = item.link);
+      default:
+        break;
+    }
+  };
+
+  return (
+    <MenuWrapper>
```

Comment:
It doesn't need to be named. I think this is left over from when I thought these would be exported primitives, but they aren't now. 

Just a side-note: The `Menu` in labs uses `Card` as its wrapper.

Reply:
I'm definitely in that camp where naming elements is confusing. If `Menu` is a component with an explicit interface, I'm okay with the added abstraction layer. If there is no added benefit to the abstraction, it should be removed.

In the future, it is possible to replace most low-level elements with `Box` for the added convenience our tokens which would be an abstraction we could understand well although it would be harder to read at first: `<Box as="ul">`. But in that case I'm not wondering what `Box` is or what element it is representing.

Reply:
Agreed that it's opaque. I'll style the `div` directly with `css`.